### ISSUE TITLE
Take your pick: Database Not found, no Mysql in Prestashop docker or Auto install not working

### ISSUE THREAD

**Issue by skygab** [PrestaShop/docker#132]
Hi!
Yes.. I have tried every single parameter and command line alternative Ive seen in readme files and help files and I don't get anything to work really... 😢 

I have a default docker installation and everything else Im working on is.. well... working.

Besides the fact that Id love to make this work and find out whats wrong... I don't understand how three documented command options, don't work as documented. Are there any parameters missing, are all this instructions and help files outdated?

So this is what happens...

## Option 1
As stated in this link and in several other places, the Docker file should have an internal database if DB_Server is not specified (https://github.com/PrestaShop/docker/issues/32#issuecomment-249601095)
But... If I try the following commands:

```shell
$ docker run -ti --name some-prestashop -p 8080:80 -d prestashop/prestashop
```

I get no DB and I can't go through the installation
The log file has no errors.. but no trace of any Mysql starting inside the local prestashop container

Log output:
```
* Reapplying PrestaShop files for enabled volumes ...

* Almost ! Starting web server now

AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.2. Set the 'ServerName' directive globally to suppress this message
[Sat Aug 04 06:22:56.238894 2018] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.25 (Debian) configured -- resuming normal operations
[Sat Aug 04 06:22:56.239027 2018] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'
```

## Option 2
So... thinking this might be outdated... I go for what it says in this current readme file at [DockerHub](https://hub.docker.com/r/prestashop/prestashop/) and here at [GitHub](https://github.com/PrestaShop/docker)
But... If I try the following commands:

```shell
$ docker run -ti --name some-mysql -e MYSQL_ROOT_PASSWORD=admin -d mysql
$ docker run -ti --name some-prestashop --link some-mysql -e DB_SERVER=some-mysql -p 8080:80 -d prestashop/prestashop
```

Both docker containers start but the prestashop container is not able to see the mysql container. 
<img width="667" alt="screen shot 2018-08-04 at 2 32 59 am" src="https://user-images.githubusercontent.com/41950758/43673310-cc305c82-978e-11e8-8035-dcdc567da10d.png">

## Option 3
So finally I go for the auto install option. There's an [issue](https://github.com/PrestaShop/docker/pull/117#issue-180671076) related to the PS_Country being in lowercase. I find this a good alternative to test as it has simple steps to reproduce the behavior and to make it work correctly.. so I think.. "hey... this might make it work!".... well... it didn't... 

```shell
$ docker run -ti --name some-mysql -e MYSQL_ROOT_PASSWORD=admin -d mysql
$ docker run -ti --name some-prestashop --link some-mysql -e DB_SERVER=some-mysql -e PS_INSTALL_AUTO=1 -e PS_COUNTRY=GB -p 8080:80 -d prestashop/prestashop
```

Log output... 
```
* Reapplying PrestaShop files for enabled volumes ...
* Waiting for confirmation of MySQL service startup
* Waiting for confirmation of MySQL service startup
* Waiting for confirmation of MySQL service startup
...
```
(Mysql Server Logs look as usual... with no apparent problems.

Help please! 😢 

## Option 4 (WORKED!)
The mods that are being applied to the current [Pull Request ](https://github.com/PrestaShop/docker/pull/129/files) work. Still there is a minor change that I had to do because of the bridging not working with Docker on Mac. I'll make the changes to the pul request.

**Basically**, you need to set the PS_DOMAIN and PS_SHOP_URL variables to localhost:8080 for it to work correctly when browsing from the host computer. The command looks something like this:
(PS_INSTALL_AUTO=1 is optional)

```bash
docker run -ti --name some-prestashop --network prestashop-net -e DB_SERVER=some-mysql -e PS_INSTALL_AUTO=1 -e PS_DOMAIN=localhost:8080 -e PS_SHOP_URL:localhost:8080 -p 8080:80 -d prestashop/prestashop
```

**Comment by bouchezb** [comment]
Hi @skygab,

Which image tag is pulled from `mysql` when you run 
```
$ docker run -ti --name some-mysql -e MYSQL_ROOT_PASSWORD=admin -d mysql
```
You can see it with command `docker images`.

I had a similar issue with mysql 8.0 and PS 1.7.4.1 which was not notifying incompatibility.

I had to force version to 5.7 to make it work

**Notice that IF YOU DO NOT MOUNT /var/www/html AS A VOLUME PS WILL AUTO INSTALL EACH TIME THE CONTAINER IS STARTED**

**Comment by skygab** [comment]
Yes.. I was using mysql 8.0 and It finally worked using option 4. Ill try the other versions now... plus, need PS 1.6 to test some stuff.  Thanks for that tip 👍

**Comment by Quetzacoalt91** [comment]
Thank you @skygab, now that #129 is merged, we can close this issue.

Thank you for your feedback.
